authzed/api/v1/permission_service.proto

```diff
@@ -406,16 +406,41 @@ message LookupSubjectsResponse {
 
   // subject_object_id is the Object ID of the subject found. May be a `*` if
   // a wildcard was found.
-  string subject_object_id = 2;
+  // deprecated: use `subject`
+  string subject_object_id = 2 [deprecated = true];
 
   // excluded_subject_ids are the Object IDs of the subjects excluded. This list
   // will only contain object IDs if `subject_object_id` is a wildcard (`*`) and
   // will only be populated if exclusions exist from the wildcard.
-  repeated string excluded_subject_ids = 3;
+  // deprecated: use `excluded_subjects`
+  repeated string excluded_subject_ids = 3 [deprecated = true];
```

Comment:
We're still going to be filling these values out right?

Reply:
Yes